exercises/practice/largest-series-product/example.cpp

```diff
@@ -0,0 +1,36 @@
+#include "example.h"
+#include <numeric>
+#include <algorithm>
+#include <stdexcept>
+#include <regex>
+namespace largest_series_product {
+
+    int largest_product(const std::string& series, int size) {
+        if (size == 0) {
+            return 1;
+        }
+
+        if(size < 0) {
+            throw std::domain_error("span must not be negative");
+        }
+
+        if ((int)series.size() < size) {
```

Comment:
Please uses a named cast instead, in this case a `static_cast`. Either `static_cast<int>(series.size()) < size` or since `size` is non-negative at that point `series.size() < static_cast<std::size_t>(size)`.